src/app/validators/custom-validators.ts

```diff
@@ -31,6 +31,30 @@ export class CustomValidators {
     };
   }
 
+  static isUniqueAnswer(uniqueRequired: boolean) {
```

Comment:
Clean the code and also it is validating choices still exam form submits and there is no message showing. Please work on that